cmd/mo-service/version.go

```diff
@@ -37,11 +37,11 @@ func maybePrintVersion() {
 		return
 	}
 
-	fmt.Println("MatrixOne build info:")
-	fmt.Printf("  The golang version used to build this binary: %s\n", GoVersion)
-	fmt.Printf("  Git branch name: %s\n", BranchName)
-	fmt.Printf("  Git commit ID: %s\n", CommitID)
-	fmt.Printf("  Buildtime: %s\n", BuildTime)
-	fmt.Printf("  Version: %s\n", Version)
+	logutil.Info("MatrixOne build info:")
```

Comment:
There is no need to change it to a log here, because it is only called when printing the version information, and the global log is not initialized at this time

Reply:
consider it done.